crates/eframe/src/native/run.rs

```diff
@@ -157,22 +179,24 @@ fn run_and_return(
                 return;
             }
 
-            // Platform-dependent event handlers to workaround a winit bug
-            // See: https://github.com/rust-windowing/winit/issues/987
-            // See: https://github.com/rust-windowing/winit/issues/1619
```

Comment:
Is this workaround no longer needed? Have you tested this on Windows?

Reply:
oh, this is now handled later instead?

Reply:
I tested on Windows VM and works fine.
And from what i heard from @stefnotch and @workingj works good!

Setting the control flow to Pull when we request a repaint, seams to fix it!